scripts/lint

```diff
@@ -1,13 +1,14 @@
 #!/bin/sh -e
 
 export PREFIX=""
-if [ -d 'venv' ] ; then
+if [ -d 'venv' ]; then
     export PREFIX="venv/bin/"
 fi
 export SOURCE_FILES="starlette tests"
 
 set -x
 
+./scripts/sync-version
```

Comment:
Should this be in `scripts/check` instead? `scripts/lint` doesn't get run by CI, but check does (mypy, flake8, black checks, etc).